data/campaigns/Under_the_Burning_Suns/_main.cfg

```diff
@@ -19,6 +19,8 @@
 
     description= _ "In the distant future a small band of elves struggles to survive amidst the ruins of fallen empires. Lead your people out of the desert on an epic journey to find a new home.
 
+" + _"Note: This campaign is probably not appropriate for beginners. It changes certain Wesnoth standards, such as the day/night cycle, recall costs and general approach to the difficulty levels, it is also making use of a custom faction of Quenoth elves as a replacement for standard elves.
```

Comment:
"... levels. The Quenoth Elves include units that can attack several times per turn, and others that can move after attacking."

Reply:
Others that use AoE attacks and change their attacks after attacking multiple times in a row. This campaign also includes heroes with often possibly not easy to use AMLAs. 

...is there even a need to go that far? I don't think so...

Reply:
If you're not going to mention anything about why the Quenoth elves are special I wouldn't mention them at all.

Reply:
I would either mention custom units and why they're special or not mention them at all.

Reply:
While the AoE could be mentioned, I think that mentioning the multiple attacks and disengage ability is enough to set the player's expectations for what the campaign will be. So I've taken just my suggestion and put in it #8056.

Reply:
Thats kind of important because you can kill your own units with it.